templates/common/_base/files/mtu-migration-conf.yaml

```diff
@@ -0,0 +1,32 @@
+{{ $target_mtu := 0 -}}
+{{ $current_mtu := 0 -}}
+{{ $cni_target_mtu := 0 -}}
+{{ if .Network }}{{ if .Network.MTUMigration -}}
+{{ if .Network.MTUMigration.Machine -}}
```

Comment:
if its possible, it would make it more readable to indent this logic. The logic looks OK to me though.